vendors/pc/boards/linux/CMakeLists.txt

```diff
@@ -6,6 +6,14 @@ if (AFR_ENABLE_UNIT_TESTS)
 
     set(CMAKE_SHARED_LINKER_FLAGS "-fPIC")
 
+    set(3rdparty_dir "${AFR_ROOT_DIR}/libraries/3rdparty")
```

Comment:
We could also use AFR_3RDPARTY_DIR, right? Then there's AFR_MODULES_*_DIR. Any reason why we're not using these variables declared in afr.cmake instead?